### ISSUE TITLE
address potential race condition in batch creation

### ISSUE THREAD

**Issue by benadida** [votingworks/module-scan#115]
```
  public async addBatch(): Promise<number> {
    await this.dbRunAsync(
      "insert into batches (startedAt) values (strftime('%s','now'))"
    )

    const { rowId } = await this.dbGetAsync(
      'select last_insert_rowid() as rowId'
    )
    return parseInt(rowId, 10)
  }
```

If two batches are added at the same time, this function may return the same ID for both.

We really want to use the autoincrementing `id` field here, but need to figure out how to return that field as part of the insert.

**Comment by eventualbuddha** [comment]
Addressed by #116?

**Comment by eventualbuddha** [comment]
Not an issue after #128
